chia/server/server.py

```diff
@@ -310,7 +318,12 @@ async def incoming_connection(self, request):
         ws = web.WebSocketResponse(max_msg_size=max_message_size)
         await ws.prepare(request)
         close_event = asyncio.Event()
-        cert_bytes = request.transport._ssl_protocol._extra["ssl_object"].getpeercert(True)
+        ssl_object = request.get_extra_info("ssl_object")
```

Comment:
I think you can directly get the peercert here with `request.get_extra_info("peercert") - https://docs.python.org/3/library/asyncio-protocol.html?highlight=peercert#asyncio.BaseTransport.get_extra_info

Reply:
Im on mobile so i cant verify but looks like then its called with `binary_form=False` and we would need to convert it?

Reply:
https://docs.python.org/3/library/ssl.html#ssl.SSLSocket.getpeercert

Reply:
Ah I see, we want `binary_form=True`, but `request.get_extra_info("peercert")` will call it as the default which is `False`. Ok resolving comment